src/test/resources/api_input/input2.json

```diff
@@ -0,0 +1,57 @@
+      ],
+      "name": "apps__c"
+    },
+    {
+      "publishedArticles": [
+        { "urlName": "can-i-read-your-papermagazines-online", "title": "Can I read your paper/magazines online?", "id": "ka05I000000UELOQA4", "dataCategories": [], "body": null },
+        { "urlName": "how-to-stay-signed-in", "title": "How to stay signed in", "id": "ka05I000000dOFAQA2", "dataCategories": [], "body": null },
+        { "urlName": "id-like-to-make-a-complaint-about-an-advertisement", "title": "I'd like to make a complaint about an advertisement", "id": "ka05I000000UC4xQAG", "dataCategories": [], "body": null },
+        { "urlName": "im-unable-to-comment-and-need-help", "title": "I'm unable to comment and need help", "id": "ka05I000000lBXmQAM", "dataCategories": [], "body": null },
+        { "urlName": "ive-found-a-bug-how-can-i-report-it", "title": "I’ve found a bug, how can I report it?", "id": "ka05I000000UC5WQAW", "dataCategories": [], "body": null },
+        { "urlName": "why-am-i-still-seeing-adsbanners", "title": "Why am I still seeing ads/banners?", "id": "ka05I000000lBY6QAM", "dataCategories": [], "body": null },
+        { "urlName": "why-have-i-been-banned-from-commenting", "title": "Why have I been banned from commenting?", "id": "ka05I000000UC4zQAG", "dataCategories": [], "body": null }
+      ],
+      "name": "website__c"
+    }
+  ],
+  "article": {
+    "urlName": "how-to-stay-signed-in",
+    "title": "How to stay signed in",
+    "id": "ka05I000000Qtw4QAC",
+    "dataCategories": [
+      { "name": "accounts__c", "label": "Accounts and sign in" },
+      { "name": "apps__c", "label": "The Guardian apps" },
+      { "name": "website__c", "label": "The Guardian website" }
+    ],
+    "body": "When you are signing into the Guardian website, we place a cookie in your browser that should recognise you when you return. This will not happen if your browser is set up in such a way as to block or delete cookies, although this will affect all of the sites that you visit. If you are getting this across multiple sites, the settings to check in your browser are:<br><br>1. Whether you are running your browser in a private or incognito mode (this happens fairly regularly for people on Safari on iOS).<br><br>2. What the cookie settings in your browser are (accepting cookies from sites you visit should cover this).<br><br>3. Whether you are running any browser extensions or add-ons that block or clear cookies to protect your privacy.<br><br>If it is only the Guardian that you are having trouble with it may be that your sign in cookie has become corrupted. We would suggest that you try clearing any Guardian cookies from your browser and signing back in.<br><br>If the above doesn’t work, or if you would like more detailed instructions on how to do any of the above, please get in touch using the contact details at the bottom of this page and note the name of the browser that you are using."
```

Comment:
No change necessary but just for my own understanding where did you get the wording for this from?

Reply:
The articles are written and stored in [Salesforce Knowledge](https://gnmtouchpoint.lightning.force.com/lightning/r/Knowledge__kav/ka05I000000Qtw4QAC/view).

 This JSON file is the request body that's sent to the API. We log the inputs for each request so you can view them here in [Cloudwatch](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logsV2:logs-insights$3FqueryDetail$3D$257E$2528end$257E0$257Estart$257E-2419200$257EtimeType$257E$2527RELATIVE$257Eunit$257E$2527seconds$257EeditorString$257E$2527fields*20*40timestamp*2c*20*40message*0a*7c*20sort*20*40timestamp*20desc*0a*7c*20limit*2020$257EisLiveTail$257Efalse$257EqueryId$257E$2527960f1831-4aae-48f7-82c7-1944f152cd51$257Esource$257E$2528$257E$2527*2faws*2flambda*2fmanage-help-content-publisher-DEV$2529$2529). I just copy/pasted the API input from there.